rbac/management/utils.py

```diff
@@ -126,6 +126,10 @@ def groups_for_principal(principal, tenant, **kwargs):
         assigned_group_set = assigned_group_set.prefetch_related(prefetch_lookups)
         platform_default_group_set = platform_default_group_set.prefetch_related(prefetch_lookups)
 
+    if kwargs.get("is_org_admin"):
+        admin_default_group_set = Group.objects.filter(admin_default=True)
```

Comment:
It's probably not totally necessary, but for consistency maybe we could add a `Group` model helper similar to `Group.platform_default_set()` here, in the event we need to reuse that filter? Also, just for explicitness, we'll probably want to add a filter to query by the public tenant only. Right now we should only ever have one default admin group, but if we change that in the future to behave like the (custom) default access group(s), it would be helpful to have this as a baseline.

Reply:
Would we want the filtering of the schema to be in the helper method?

Reply:
I think where you've got it makes sense, and follows the pattern we have.